### ISSUE TITLE
Is there possibility to create other output options?

### ISSUE THREAD

**Issue by MABlosfeld** [opentrack/opentrack#1302]
Keyboard or Joystick for example.

**Comment by sthalik** [comment]
There's already a joystick output module. Keyboard really isn't feasible.

**Comment by sthalik** [comment]
Oops, closed by accident.

**Comment by MABlosfeld** [comment]
Yes I know but when trying to use the vJoystick output option it conflicts with the joystick virtual control application. There is the MOUSE EMULATION output option and it works as it should, I believe if there were the JOYSTICK's EMULATION output option (not VJOY) it would work without conflicts.
